### ISSUE TITLE
Reset Normalize_y = True

### ISSUE THREAD

**Issue by sqbl** [novonordisk-research/ProcessOptimizer#8]
normalize_y is temporarily set to "=False" to solve issue in #5 

When SKLEARN have implemented changes as discussed in https://github.com/scikit-learn/scikit-learn/pull/18388, we can reset the normalize_y to the old value of True

**Comment by sqbl** [comment]
Chenges were made to normalize_y=False. This is a temporary fix until sklearn implements solution
https://github.com/novonordisk-research/ProcessOptimizer/pull/9
